pkg/storage/buckets/store.go

```diff
@@ -54,30 +56,8 @@ type Meta struct {
 }
 
 // NewStore instantiates BucketStore
-func NewStore(stream streams.Store) Store {
-	// root object store for storing the buckets with unencrypted names
-	store := objects.NewStore(stream, storj.Unencrypted)
-	return &BucketStore{store: store, stream: stream}
-}
-
-// GetObjectStore returns an implementation of objects.Store
-func (b *BucketStore) GetObjectStore(ctx context.Context, bucket string) (objects.Store, error) {
-	if bucket == "" {
-		return nil, storj.ErrNoBucket.New("")
-	}
-
-	m, err := b.Get(ctx, bucket)
```

Comment:
I think we might need to call b.Get() in order to get the `PathEncryptionType`.  It looks like the `streams.Cipher` is trying to be used here instead (re: https://github.com/storj/storj/pull/1854/files#diff-19374caab19525948168b09d2e0f94aaR166), but I think  streams.cipher is used for data encryption, not for the pathCipher